doc/source/whatsnew/v1.1.0.rst

```diff
@@ -98,6 +98,9 @@ Performance improvements
 Bug fixes
 ~~~~~~~~~
 
+**Index**
```

Comment:
if you are going to make another label then follow the current style (^^^^). But just easier to put this in reshaping